include/grid_map_geo/grid_map_geo.hpp

```diff
@@ -39,11 +39,20 @@
 #include <grid_map_core/GridMap.hpp>
 #include <grid_map_core/iterators/GridMapIterator.hpp>
 
+#if __APPLE__
```

Comment:
If we have platform specific code in this library, how do you plan to maintain it? 

My thought is to add a MacOS CI job.

Reply:
Sure - for follow up. macOS is tier 3, community support with no guarantees ;-)